scripts/requirements.txt

```diff
@@ -4,3 +4,8 @@ numpy==1.18.5
 pandas==1.0.4
 requests==2.24.0
 urllib3==1.25.9
+datacommons==1.2.0
+geojson==2.5.0
+geopandas==0.8.1
+matplotlib==3.3.0
+rdp==0.8
```

Comment:
Let's alphabetize these for easier human reading.

Reply:
Done.